apps/exceptions.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+class UrlShortenerError(APIException):
+    status_code = status.HTTP_503_SERVICE_UNAVAILABLE
+    default_detail = 'Internal Service Error.'
+    default_code = 'server_error'
+
+    def __init__(self, detail, status_code=None, code=None):
+        super(UrlShortenerError, self).__init__(detail, code)
```

Comment:
In python3.x `super().__init__(detail, status_code=status_code, code=code)` is sufficient.